test/fixtures/library.j/main/src/library/j/dependency.js

```diff
@@ -0,0 +1,53 @@
+/*!
+ * ${copyright}
+ */
+sap.ui.define([], function () {
+	sap.ui.getCore().initLibrary({
+		name: "testlib",
```

Comment:
Hopefully, now should be a bit better. I thought that, although those libraries are not truly needed, it might be 
 for good to have them with the ENUMs defined for the sake of clarity, readability, and for a better understanding of the code. The scenario with _sap.ui.core.libary_ and _sap.m.library_ inside a control, for example, seems to be a valid one. Though you're quite correct, and there isn't a place in UI5 where we have multiple library.js files defined in a single "library"